TheMapCloudWebServices/mapcloud_authentication.py

```diff
@@ -0,0 +1,109 @@
+from urllib2 import HTTPError, URLError
+from PyQt4.QtCore import QSettings
+from PyQt4.QtGui import QMessageBox
+
+from osma_web_services_dialog import MapcloudAuthDialog
+from layers import BASE_API_URL, make_mapcloud_request
+
+__author__ = 'matthew.walsh'
+
+
+class MapCloudAuthentication:
+
+    username = None
+    password = None
+
+    def __init__(self, iface):
+
```

Comment:
Saw a QGIS crash when this dialog was shown after a reset. Not sure what happened